[2019-08-27 21:24:27,499] {taskinstance.py:616} INFO - Dependencies all met for <TaskInstance: stock_data.upload_to_s3 2019-08-28T02:20:30.889081+00:00 [queued]>
[2019-08-27 21:24:27,507] {taskinstance.py:616} INFO - Dependencies all met for <TaskInstance: stock_data.upload_to_s3 2019-08-28T02:20:30.889081+00:00 [queued]>
[2019-08-27 21:24:27,507] {taskinstance.py:834} INFO - 
--------------------------------------------------------------------------------
[2019-08-27 21:24:27,507] {taskinstance.py:835} INFO - Starting attempt 1 of 2
[2019-08-27 21:24:27,508] {taskinstance.py:836} INFO - 
--------------------------------------------------------------------------------
[2019-08-27 21:24:27,519] {taskinstance.py:855} INFO - Executing <Task(PythonOperator): upload_to_s3> on 2019-08-28T02:20:30.889081+00:00
[2019-08-27 21:24:27,519] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'stock_data', 'upload_to_s3', '2019-08-28T02:20:30.889081+00:00', '--job_id', '9', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/hello_world_dag.py', '--cfg_path', '/var/folders/z6/sk1b92g1385cmsp5qvg672kr0000gn/T/tmparg9uxvl']
[2019-08-27 21:24:28,513] {base_task_runner.py:115} INFO - Job 9: Subtask upload_to_s3 [2019-08-27 21:24:28,512] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-08-27 21:24:28,926] {base_task_runner.py:115} INFO - Job 9: Subtask upload_to_s3 [2019-08-27 21:24:28,925] {dagbag.py:90} INFO - Filling up the DagBag from /Users/hdeva/airflow/dags/hello_world_dag.py
[2019-08-27 21:24:34,242] {base_task_runner.py:115} INFO - Job 9: Subtask upload_to_s3 [2019-08-27 21:24:34,241] {cli.py:516} INFO - Running <TaskInstance: stock_data.upload_to_s3 2019-08-28T02:20:30.889081+00:00 [running]> on host Haris-MacBook-Pro.local
[2019-08-27 21:24:39,252] {python_operator.py:105} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=stock_data
AIRFLOW_CTX_TASK_ID=upload_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2019-08-28T02:20:30.889081+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2019-08-28T02:20:30.889081+00:00
[2019-08-27 21:24:39,368] {logging_mixin.py:95} INFO - [[34m2019-08-27 21:24:39,367[0m] {[34mcredentials.py:[0m1182} INFO[0m - Found credentials in shared credentials file: [1m~/.aws/credentials[0m[0m
[2019-08-27 21:24:40,385] {python_operator.py:114} INFO - Done. Returned value was: None
[2019-08-27 21:24:42,587] {logging_mixin.py:95} INFO - [[34m2019-08-27 21:24:42,585[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 0[0m
